src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  const pricePerDays = days * basePrice;
+
+  if (days >= 7) {
+    return pricePerDays - 50;
```

Comment:
It's better not to use mystery numbers and create variables for 20 and 50 values as well

Reply:
Thank you, Anastasiia. I will note it for the following tasks.